cmd/mtchannel_broker/main.go

```diff
@@ -19,14 +19,21 @@ package main
 import (
 	// Uncomment the following line to load the gcp plugin (only required to authenticate against GKE clusters).
 	// _ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	"flag"
 
 	"knative.dev/pkg/injection/sharedmain"
+	"knative.dev/pkg/signals"
```

Comment:
```suggestion
	"knative.dev/pkg/signals"
	"k8s.io/client-go/rest"
```

Reply:
What's this about? is it parsing the comment about rest.config??

Reply:
This isn't sockpuppet, my suggestion above referenced constants from this package and I was nice and suggested the appropriate import too 🤣 

![image](https://user-images.githubusercontent.com/2442466/87859641-873ccd00-c8eb-11ea-9fa0-a1ab7f15dac9.png)